Tasks/day.js

```diff
@@ -1,15 +1,11 @@
 // Get day number
+'use strict';
 
-let D = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
+const days = ['sunday', 'monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday', ];
 
-_parse_day_ = (s) => {
-  let i;
-  for (i = 0; i < D.length; i++) {
-    if (s.startsWith(D[i].toLowerCase())) {
-      return i + 1;
-    }
-  }
-  return -1;
-};
+const day = (s) => {
+    const hit = days.indexOf(s);
+    return hit +1 || -1;
+    };
```

Comment:
We have certain linter config for auto format code style

```suggestion
const day = (s) => {
  const hit = days.indexOf(s);
  return hit + 1 || -1;
};
```